lk2nd/device/dts/msm8952/msm8937-mtp.dts

```diff
@@ -15,6 +15,14 @@
 		lk2nd,match-device = "cedric";
 
 		lk2nd,dtb-files = "msm8937-motorola-cedric";
+		
+		lk2nd,match-panel;
+  		panel {
+   			compatible = "motorola,cedric-panel", "lk2nd,panel";
+   			qcom,mdss_dsi_mot_tianma_497_1080p_video_v0 {
+    				compatible = "motorola,cedric-nt35596-tianma";
+   			};
+  		};
```

Comment:
```suggestion

		lk2nd,match-panel;

		panel {
			compatible = "motorola,cedric-panel", "lk2nd,panel";

			qcom,mdss_dsi_mot_tianma_497_1080p_video_v0 {
				compatible = "motorola,cedric-nt35596-tianma";
			};
		};
```

- Drop trailing white spaces.
- Use tabs for indents.
- Some newline nitpicks by the way.